queries.sql

```diff
@@ -17,3 +17,20 @@ SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 AND 17.30;
 
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, sum(escape_attempts) FROM animals
+GROUP BY neutered;
+
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
+GROUP BY species;
+
+SELECT species, AVG(escape_attempts) FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
```

Comment:
- [ ] In this one you should do '2000-31-12' instead of '2000-01-01' because the year 2000 should include December.  Kindly correct this one.